environment/deployments/science-platform/env/dev-cloudsql.tfvars

```diff
@@ -3,20 +3,42 @@ environment      = "dev"
 application_name = "science-platform"
 project_id       = "science-platform-dev-7696"
 
-# Butler database
-butler_db_name     = "butler-registry-dev"
-butler_require_ssl = false
-butler_database_flags = [
+# Butler Registry Original Database
+butler_registry_db_name          = "butler-registry-dev"
+butler_registry_database_version = "POSTGRES_13"
+butler_registry_require_ssl      = false
+butler_registry_database_flags = [
   { name = "password_encryption", value = "scram-sha-256" }
 ]
-butler_database_version = "POSTGRES_13"
-butler_ipv4_enabled     = true
+butler_registry_ipv4_enabled                           = true
+butler_registry_db_maintenance_window_day              = 1
+butler_registry_db_maintenance_window_hour             = 23
+butler_registry_db_maintenance_window_update_track     = "canary"
+butler_registry_backups_enabled                        = true
+butler_registry_backups_point_in_time_recovery_enabled = true
```

Comment:
Since all these variables got renamed, won't we also need these same changes in `integration-cloudsql.tfvars` and `production-cloudsql.tfvars`?